### DISCUSSION TITLE
Getting pywin32 excelAddin.py working on Windows 7 Microsoft Excel 2010

### DISCUSSION THREAD

**Discussion by caoyudeng** [mhammond/pywin32#2549] [General]
\Lib\site-packages\win32com\demos\excelAddin.py invalid.

Though the Com add-in registers itself successfully and is displayed in "com add-ins" list in Excel, it does not work.

Only error feedback I get is on the "Load behavior" line in excel's "com add-ins" dialog, says "Not loaded. A runtime error occurred during the loading of the com add-in"

How can I find out what goes wrong?

Version: 
python2.7/python3.8
pywin32
excel 2010 32bit/64bit
win7

**Comment by mhammond** [comment]
If you installed via pypi, it's probably due to the fact the post-install scripts haven't run - try installing from the .exe installer. This is more a support request than a bug request, so you should email the pywin32 mailing list as the issue template mentioned - but feel free to reopen if you can identify a clear bug.
